src/patternfly/components/InputGroup/styles.scss

```diff
@@ -11,16 +11,6 @@
   --pf-c-input-group__text--BorderSize: var(--pf-global--BorderWidth--sm);
```

Comment:
L6 & L8 (`--pf-c-input-group__text--PaddingTop` and `--pf-c-input-group__text--PaddingBottom`) can be removed.